airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -729,66 +735,34 @@ def trigger_reentry(self, context: Context, event: dict[str, Any]) -> Any:
 
             if self.do_xcom_push:
                 result = self.extract_xcom(pod=self.pod)
-            remote_pod = self.pod_manager.await_pod_completion(self.pod)
+                if event["status"] in ("error", "failed", "timeout"):
+                    raise AirflowException(event)
```

Comment:
Can we have here:
```
if "stack_trace" in event:
    message = f"{event['message']}\n{event['stack_trace']}"
else:
    message = event["message"]
raise AirflowException(message)
```

to be consistent with earlier `execute_complete`

Reply:
I think we already have better traceback formatting in the triggerer. So this may not be needed.